modules/container-registry/registries/main.bicep

```diff
@@ -311,6 +314,14 @@ module registry_replications 'replications/main.bicep' = [for (replication, inde
   }
 }]
 
+module registry_caches 'cache-rules/main.bicep' = {
```

Comment:
After changing child parameter input, please iterate on array here and pass individual object settings of each element. You can refer to `registry_webhooks` below as an example